src/components/Section.js

```diff
@@ -6,9 +6,7 @@ export default class Section {
   }
 
   renderItems() {
-    this._renderedItems.forEach((item) => {
-      this._renderer(item);
-    });
+    this._renderedItems.forEach(this._renderer);
```

Comment:
Needs correction: The items are not already rendered, so you should rename `_renderedItems` variable to `_items`